src/realizations/catchment/Bmi_Module_Formulation.cpp

```diff
@@ -311,8 +312,8 @@ namespace realization {
             if( !bmi_var_name.empty() )
             {
                 //Get forcing value from BMI variable
-                double value = get_var_value_as_double(0, bmi_var_name);
-
+                double value = get_var_value_as_double(output_index, bmi_var_name);
+                
```

Comment:
Github is showing that you added a bunch of indentation whitespace on this blank line.